src/client/components/dashboard/robot_panel/view_model.ts

```diff
@@ -0,0 +1,75 @@
+import { computed } from 'mobx'
+import { createTransformer } from 'mobx'
+import { message } from '../../../../shared/proto/messages'
+import { DashboardRobotModel } from '../dashboard_robot/model'
+import { LastStatus } from './view'
+import State = message.behaviour.Behaviour.State
+
+import Mode = message.input.GameState.Data.Mode
+import PenaltyReason = message.input.GameState.Data.PenaltyReason
+import Phase = message.input.GameState.Data.Phase
+
+export class RobotPanelViewModel {
+  public constructor(private model: DashboardRobotModel) {
+  }
+
+  public static of = createTransformer((model: DashboardRobotModel) => {
+    return new RobotPanelViewModel(model)
+  })
+
+  @computed
+  public get batteryValue(): string {
+    const battery = this.model.battery
+    return battery === -1 ? '' : `${Math.round(battery * 100)}%`
+  }
+
+  @computed
+  public get behaviour(): string {
+    return State[this.model.behaviourState] || State[State.UNKNOWN]
+  }
+
+  @computed
+  public get lastCameraImage(): LastStatus {
+    return this.getLastStatus(this.model.lastCameraImage, 3)
+  }
+
+  @computed
+  public get lastSeenBall(): LastStatus {
+    return this.getLastStatus(this.model.lastSeenBall, 5)
```

Comment:
Depends what you mean by configuration, we don't have standalone configuration files but it could be part of the model which is what we currently use for configuration.

@MonicaOlejniczak I'd say for now set it to 30 seconds for ball/goals, and 5 seconds for camera. Can we also display the time since seen in title text? (with that fancy tooltip we were talking about as a follow-up?)